x-pack/plugins/cases/public/components/create/template.test.tsx

```diff
@@ -53,7 +53,63 @@ describe.skip('CustomFields', () => {
     );
 
     await waitFor(() => {
-      expect(onTemplateChange).toHaveBeenCalledWith(selectedTemplate.caseFields);
+      expect(onTemplateChange).toHaveBeenCalledWith({
+        caseFields: selectedTemplate.caseFields,
+        key: selectedTemplate.key,
+      });
+    });
+  });
+
+  it('shows selected template as default', async () => {
```

Comment:
as this whole suite is skipped, what about adding this tests in a separated/non-skipped describe? I would be a shame that something is broken and we don't notice because we added them here. Also, moving them will reduce help keeping the noise as low as possible when trying to fix the flakiness

Reply:
This test suite is already un-skipped and afterEach was added by [clear cache PR](https://github.com/elastic/kibana/pull/191212/files#diff-5c4e6ae25f523f9068fc9206433db412001735d35852d9d22aada2ca98e74fef). So I didn't create separate describe for new changes.